src/avram/model.cr

```diff
@@ -72,6 +73,7 @@ abstract class Avram::Model
   macro view(view_name = nil)
     {{ yield }}
 
+    validate_columns
```

Comment:
I considered moving this macro into the `setup` list below but if I did that it would have added a layer to the error message that pointed at macro code so I thought this makes the error easier to understand